packages/visx-demo/src/components/Gallery/index.tsx

```diff
@@ -15,6 +15,8 @@ import * as BarsTile from './BarsTile';
 import * as BrushTile from './BrushTile';
 import * as ChordTile from './ChordTile';
 import * as CurvesTile from './CurvesTile';
+import * as DelaunayTile from './DelaunayTriangulationTile';
+import * as DelaunayVoronoiTile from './DelaunayVoronoiTile';
```

Comment:
I wonder if it'd be worth removing the (now legacy) `voronoi` tile from this list

Reply:
Good idea 👍